### ISSUE TITLE
[Feature Request]:  Create automatic Launch.json settings for  (VSC-995)

### ISSUE THREAD

**Issue by Panometric** [espressif/vscode-esp-idf-extension#827]
**Is your feature request related to a problem? Please describe.**
Getting the idf and vscode platform to run a debugger on a fresh install is one of the most frustrating I have ever used. 
Among other things, having to paste some text into launch.json that is only shown in [Debugging.md](https://github.com/espressif/vscode-esp-idf-extension/blob/81cc28e0f90dc3aadeeb0060f88e06583e380622/docs/DEBUGGING.md) was difficult to find. 
Most other platforms will suggest working values that work without additional interaction.  It's great that the espidf type is supported and the parameters are known, but a new user can't easily find that. 

**Describe the solution you'd like**
Ideally, then the extension configures for a project, or the device is selected, the launch.json should be populated if empty.  And the Add configuration button on launch,json should show ESP IDF, and add the minimal values when selected.  

**Describe alternatives you've considered**
There don't seem to be any easy button.  you must find it  in the docs and paste it in manually. 

**Additional context**
This is the initial dialog on a freshly imported project:
![image](https://user-images.githubusercontent.com/716218/198158534-0ccf01c8-9c8f-4d11-85d4-5d7131240a15.png)
Manually pressing add gets this far, but no esp-idf settings are in this list. 
![image](https://user-images.githubusercontent.com/716218/198158620-8d9df398-81ec-47cd-8d2f-6af6878c40fa.png)

Selecting espidf should insert this (renamed for clarity):
```
{
    "version": "0.2.0",
    "configurations": [
        {
            "type": "espidf",
            "name": "ESP Debug",
            "request": "launch",
        }
    ]
}
```

**Comment by brianignacio5** [comment]
Isn't the `ESP-IDF: Add vscode configuration folder` doing what you are requesting ?

These files are added when you create a project with the `ESP-IDF: Show Examples projects` and `ESP-IDF: New Project`

When you click create a debug launch.json file:
![Screen Shot 2022-10-27 at 11 07 14 AM](https://user-images.githubusercontent.com/6033246/198198054-1204d90e-2045-4bd4-9a5c-c600809603d5.png)

I'll add the fix for the IntelliSense autocomplete in #825

**Comment by Panometric** [comment]
Yes,  `ESP-IDF: Add vscode configuration folder` would work. I did not know it was there. 

I think my problem was that they aren't if you are opening an exiting project, like examples from https://github.com/espressif/esp-box/tree/master/examples 

If you try one of these, when the esp-idf configures when opening a project folder for the first time, it does create settings.json,  but it does not create launch.json. 

If it did `ESP-IDF: Add vscode configuration folder` so that the configuration was complete, it would work as expected.
